sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/api/Azure.ResourceManager.SpringAppDiscovery.netstandard2.0.cs

```diff
@@ -0,0 +1,414 @@
+        public int? InstanceCount { get { throw null; } set { } }
+        public int? JvmMemoryInMB { get { throw null; } set { } }
+        public string MachineArmId { get { throw null; } set { } }
+    }
+    public partial class SpringBootAppMiscsItem
+    {
+        public SpringBootAppMiscsItem(string key) { }
+        public string Key { get { throw null; } set { } }
+        public string Value { get { throw null; } set { } }
+    }
+    public partial class SpringBootAppPatch : Azure.ResourceManager.Models.ResourceData
+    {
+        public SpringBootAppPatch() { }
+        public Azure.ResourceManager.SpringAppDiscovery.Models.SpringBootAppProperties Properties { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
+    }
+    public partial class SpringBootAppProperties
+    {
+        public SpringBootAppProperties() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.SpringAppDiscovery.Models.SpringBootAppApplicationConfigurationsItem> ApplicationConfigurations { get { throw null; } }
+        public string AppName { get { throw null; } set { } }
+        public int? AppPort { get { throw null; } set { } }
+        public string AppType { get { throw null; } set { } }
+        public string ArtifactName { get { throw null; } set { } }
+        public System.Collections.Generic.IList<int> BindingPorts { get { throw null; } }
+        public string BuildJdkVersion { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> Certificates { get { throw null; } }
+        public string Checksum { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> ConnectionStrings { get { throw null; } }
+        public System.Collections.Generic.IList<string> Dependencies { get { throw null; } }
+        public System.Collections.Generic.IList<string> Environments { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.SpringAppDiscovery.Models.SpringBootSiteError> Errors { get { throw null; } }
+        public int? InstanceCount { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.SpringAppDiscovery.Models.SpringBootAppInstancesItem> Instances { get { throw null; } }
+        public string JarFileLocation { get { throw null; } set { } }
+        public int? JvmMemoryInMB { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> JvmOptions { get { throw null; } }
+        public System.DateTimeOffset? LastModifiedOn { get { throw null; } set { } }
+        public System.DateTimeOffset? LastUpdatedOn { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> MachineArmIds { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.SpringAppDiscovery.Models.SpringBootAppMiscsItem> Miscs { get { throw null; } }
```

Comment:
spell check is reporting this word `miscs`.
We could either ignore it by adding it to the exception list,
or we could change the name of this property to expand it to a proper word.